02-express-tutorial/answers/13-router-people.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const router = express.Router()
+
+const {
+  getPeople,
+  createPerson,
+  createPersonPostman,
+  updatePerson,
+  deletePerson,
+} = require('../controllers/people')
+
+// router.get('/', getPeople)
+// router.post('/', createPerson)
+// router.post('/postman', createPersonPostman)
+// router.put('/:id', updatePerson)
+// router.delete('/:id', deletePerson)
+
+router.route('/').get(getPeople).post(createPerson)
+router.route('/postman').post(createPersonPostman)
+router.route('/:id').put(updatePerson).delete(deletePerson)
+
+module.exports = router
```

Comment:
Since you have this file under `routes/people.js` it doesn't need to be copied here as well.